meteor-app/imports/api/graphql/schema.graphql

```diff
@@ -269,6 +270,8 @@ type Review {
 	comments(pageNum: Int = 0, pageSize: Int = 24): [Comment!]!
 	"The vote that the currently logged in user cast on this review."
 	currentUserVote: Vote
+	"The id of the user that referred the user that wrote this review."
+	referredBy: String
```

Comment:
Why are you exposing this information on the API? I think this should be kept private.